docs/group.md

```diff
@@ -11,7 +19,11 @@
 
 ## Create group
 
-To create a new group, use {meth}`api.group.create`.
+Groups are created with {meth}`api.group.create` providing a group id.
+
+```{todo}
+Why is the api parameter for the id named 'groupname'?
+```
```

Comment:
Wild guess: "ID" is usually a positive integer, whereas "name" usually is a string. Hence "groupname" instead of "groupid".